### ISSUE TITLE
Network prototxt in SphereFace

### ISSUE THREAD

**Issue by kalyo-zjl** [wy1iu/sphereface#5]
Hi, @wy1iu @ydwen 

Thank you for sharing the code. I have trained a model using 28layer (as the center face) with A-softmax, which works well. However, I have struggled training with the 64layer structure you mentioned in the original paper for several days. Could you please kindly release one of you network architectures in the paper (either 20layer or 36layer or 64layer) so I can compare the difference between the prototxt I wrote from yours. 

BTW, it seems there is no pooling layer in your architecture, right? Just using convolution layer of stride 2 instead of pooling?

Thanks in advance!

**Comment by wy1iu** [comment]
The paper has described the details of the network. It should be self-contained.
